VERSION

```diff
@@ -1 +1 @@
-0.2.3
\ No newline at end of file
+0.2.3-local-jquery-ui
```

Comment:
Can we leave this at 0.2.3? I'll do a new release once we merge the PR, but changing this here will upset the release system.

Reply:
Yep! Sorry about that, I committed that in my repository but it accidentally went to PR as well :/ My first PR :P

Reply:
No worries, and congratulations on your first pull request! It is very much appreciated :-)